src/as2fm/jani_generator/scxml_helpers/scxml_tags.py

```diff
@@ -645,6 +653,12 @@ def write_model(self):
                 if transition_event not in self._event_to_conditions:
                     self._event_to_conditions[transition_event] = []
                 self._event_to_conditions[transition_event].append(transition_condition)
+        # if "" in self._events_no_condition, then we can transition to new states without events
```

Comment:
This should address what I was mentioning in https://github.com/convince-project/AS2FM/issues/70
We can discuss it on Monday.